### DISCUSSION TITLE
[Feature] Dedicated Memories Section

### DISCUSSION THREAD

**Discussion by andriykrefer** [immich-app/immich#11016] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The current "moments" feature is already a feature that I love. It shows other media with the same date on previous years.
My suggestion is to expand on that and, like the competitor app, add smarter grouping logic like:

- Recognize trips (usually a couple of day) (may be recognized by GPS location) 
- Groups photos from one person
- Holidays
- Preference to favorited media
- Create a timeline of moments events

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2836/#7437/#8654

Please use the search like the template requests.
